src/main/java/com/couchbase/lite/support/RemoteMultipartRequest.java

```diff
@@ -44,7 +44,7 @@ public void run() {
             throw new IllegalArgumentException("Invalid request method: " + method);
         }
 
-        request.addHeader("Accept", "*/*");
+        request.addHeader("Accept", "*/*"); // should be "application/json"??
```

Comment:
I think it's `*/*` because the server might return `multipart/related` instead of `application/json`